### PR TITLE
Translate to Portuguese: examples/mp-jsonb-configuration

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira: https://issues.apache.org/jira/browse/TOMEE-2830

### CODE REVIEW

File: examples/mp-jsonb-configuration/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    }
+
+}
+----
+
+`JsonbConfig` oferece muitas configurações.
+
+== Acessando o Recurso Rest
+
+O teste ativa um aplicativo da web openejb e chama o restante do recurso `/users`
```
Comment (hbelmiro) [inline examples/mp-jsonb-configuration/README_pt.adoc:130]:
```suggestion
O teste ativa um aplicativo da web openejb e invoca o endpoint `/users`
```

File: examples/mp-jsonb-configuration/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+        Assert.assertTrue(message.contains(sdf.format(new Date())));
+        // test withFormatting(true)
+        Assert.assertTrue(message.contains(System.getProperty("line.separator")));
+    }
+
+}
+----
+
+== Executando
+Executando o exemplo se pode fazer desde maven com um simples comando `mvn clean install`, executando desde o diretorio `mp-jsonb-configuration`.
```
Comment (hbelmiro) [inline examples/mp-jsonb-configuration/README_pt.adoc:177]:
```suggestion
A execução do exemplo pode ser feita a partir do maven com um simples comando `mvn clean install`, executando no diretorio `mp-jsonb-configuration`.
```

File: examples/mp-jsonb-configuration/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+
+----
+
+=== Dentro do jar
+
+`javaee-api: 8.0` ele incorpora todas as dependências necessárias para ativar um aplicativo REST em funcionamento.
+
+Se olharmos para o jar construído por maven, veremos que o aplicativo em si é bem pequeno:
```
Comment (hbelmiro) [inline examples/mp-jsonb-configuration/README_pt.adoc:283]:
```suggestion
Se olharmos para o jar construído pelo maven, veremos que o aplicativo em si é bem pequeno:
```

File: examples/mp-jsonb-configuration/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+  1701 Fri Dec 28 19:36:06 CET 2018 WEB-INF/classes/org/superbiz/JSONBConfiguration.class
+  1224 Fri Dec 28 18:28:32 CET 2018 WEB-INF/web.xml
+     0 Fri Dec 28 19:36:10 CET 2018 META-INF/maven/
+     0 Fri Dec 28 19:36:10 CET 2018 META-INF/maven/org.superbiz/
+     0 Fri Dec 28 19:36:10 CET 2018 META-INF/maven/org.superbiz/mp-jsonb-configuration/
+  1791 Fri Dec 28 19:10:44 CET 2018 META-INF/maven/org.superbiz/mp-jsonb-configuration/pom.xml
+   128 Fri Dec 28 19:36:08 CET 2018 META-INF/maven/org.superbiz/mp-jsonb-configuration/pom.properties
+----
+
+Esse jar individual pode ser implementado em qualquer implementação Java EE que atenda aos requisitos. No TomEE, basta colocá-lo no diretório `${tomee.home}/webapps/`.
```
Comment (hbelmiro) [inline examples/mp-jsonb-configuration/README_pt.adoc:308]:
```suggestion
Esse jar individual pode ser implantado em qualquer implementação Java EE que atenda aos requisitos. No TomEE, basta colocá-lo no diretório `${tomee.home}/webapps/`.
```

File: examples/mp-jsonb-configuration/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+:jbake-status: status=published
+= Configurando JSON-B
+
+Este exemplo mostra como personalizar jsonb para uma aplicação JAX-RS.
+JSONB é o novo padrão `javaee-api: 8.0` para serialização/desserialização de json. 
+Poucas anotações são necessárias e o `JsonbConfig` oferece muitas configurações.
+
+== Executando e testando o Endpoint
+
+A aplicação pode ser executada com `mvn clean install tomee: run` e se a porta 8080 estiver disponivel,pode invocar o seguinte endpoint:
```
[the remark this answers is not in the document]
Comment (Daniel-Dos) [reply examples/mp-jsonb-configuration/README_pt.adoc]: done thanks

File: examples/mp-jsonb-configuration/README_pt.adoc (hunk 2 above)
Comment (Daniel-Dos) [reply examples/mp-jsonb-configuration/README_pt.adoc]: done thanks

File: examples/mp-jsonb-configuration/README_pt.adoc (hunk 4 above)
Comment (Daniel-Dos) [reply examples/mp-jsonb-configuration/README_pt.adoc]: done thanks

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro for review.

**Pull request merged**